src/filesystem/interface.js

```diff
@@ -206,6 +206,60 @@ function FileSystem(options, callback) {
     return watcher;
   };
 
+  //Object that uses filenames as keys
+  const statWatchers = new Map();
+
+  this.watchFile = function (filename, options, listener) {
+    let prevStat, currStat;
```

Comment:
We should probably call `Path.normalize(filename)` on `filename` before we use it internally.